ontology/cio.md

```diff
@@ -2,8 +2,8 @@
 layout: ontology_detail
 id: cio
 contact:
-  email: bgee@sib.swiss
-  label: Frederic Bastian
+  label: Frédéric Bastian
+  email: frederic.bastian@unil.ch
```

Comment:
Seems to be missing the github handle..

Reply:
Interestingly, this one already had the right github handle annotated so it doesn't appear in this PR's diff